### PR TITLE
mmark: init at 1.3.6

### PR DESCRIPTION
**Pull request by yrashk**
###### Motivation for this change

mmark is not found in Nixpkgs as a standalone binary (only found as a Go dependency for `hugo`)

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/typesetting/mmark/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "mmark-${version}";
+  version = "1.3.6";
+  rev = "v${version}";
+
+  goPackagePath = "github.com/miekg/mmark";
+
+  src = fetchgit {
```
Comment (nlewo) [inline pkgs/tools/typesetting/mmark/default.nix]: Could you use `fetchFromGithub`?

File: pkgs/tools/typesetting/mmark/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
```
Comment (nlewo) [inline pkgs/tools/typesetting/mmark/default.nix]: There are useless attributes.

**Review by nlewo — CHANGES_REQUESTED** [review]

**Comment by yrashk** [comment]
@nlewo any better now?

**Comment by nlewo** [comment]
@yrashk yes, thx.
@GrahamcOfBorg build mmark

**Pull request merged**